c_glib/test/test-chunked-array-datum.rb

```diff
@@ -49,7 +49,14 @@ def test_false
   end
 
   def test_to_string
-    assert_equal("ChunkedArray([\n" + "  [\n" + "    true,\n" + "    false\n" + "  ]\n" + "])", @datum.to_s)
+    assert_equal(<<-CHUNKED_ARRAY.chomp, @datum.to_s)
```

Comment:
This is not chunked array. This is chunked array datum. So `DATUM` (or `CHUNKED_ARRAY_DATUM`) is better than `CHUNKED_ARRAY`.

Reply:
Thanks! Changed to `DATUM`.